lldb/include/lldb/API/SBProcess.h

```diff
@@ -398,6 +398,13 @@ class LLDB_API SBProcess {
   /// valid.
   lldb::SBProcessInfo GetProcessInfo();
 
+  /// Return target dump file during postmortem debugging.
+  /// An empty file will be returned for live debugging.
+  ///
+  /// \return
+  ///     The target dump file spec.
+  lldb::SBFileSpec GetCoreFile();
+
```

Comment:
We load the core file in SBTarget. I would move this into SBTarget.h/.cpp

Reply:
@clayborg, I agree with what @jasonmolenda says here. If you look at the existing implementation, the core file passed in is not stored in Target object but inside the created process object. `SBTarget::GetCoreFile()` would require us redundantly store the core file path inside target.